Makefile

```diff
@@ -118,26 +118,23 @@ ifneq (${BUILD_PLATFORM},aarch64)
 endif
 
 test:
-	${GOTEST} -tags unit -coverprofile cover.out -timeout=60s ./agent/...
-	go tool cover -func cover.out > coverprofile.out
+	@cd agent && $(MAKE) test
 
 test-silent:
-	$(eval VERBOSE=)
-	${GOTEST} -tags unit -coverprofile cover.out -timeout=60s ./agent/...
-	go tool cover -func cover.out > coverprofile.out
+	@cd agent && $(MAKE) test-silent
 
 .PHONY: analyze-cover-profile
-analyze-cover-profile: coverprofile.out
+analyze-cover-profile: ./agent/coverprofile.out
 	./scripts/analyze-cover-profile
 
 run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tests
-	ECS_LOGLEVEL=debug ${GOTEST} -tags integration -timeout=30m ./agent/...
+	@cd agent && ECS_LOGLEVEL=debug GO111MODULE=on ${GOTEST} -tags integration -timeout=30m ./...
```

Comment:
this is more out of ignorance, but wouldn't the working dire be messed up if two targets are executed sequentially? say `run-integ-tests`, which invokes `run-sudo-tests`, so there is a `cd agent`, then once we're in `agent` folder, there would be another `cd agent`, which would not exist?

Reply:
these two targets run in different shell so directory will not be messed up in this case.